core/create/src/prompts/options.ts

```diff
@@ -44,7 +45,8 @@ async function optionsPromptForPlugin(
             {
               name: 'stringOption',
               type: 'text',
-              message: `Set a value for '${styles.option(optionName)}'` + defaultSuffix
+              message:
+                `Set a value for '${styles.option(optionName)}'` + ` (${typeDescription})` + defaultSuffix
```

Comment:
**issue**: we should be adding the `typeDescription` string into all the different prompt types as a hint, not just prompts that accept a string